next/site-config/anvil-portal/dev/navigation.ts

```diff
@@ -6,6 +6,15 @@ import { NavigationConfig } from "../../../docs/common/entities";
 
 // Site map for the MDX pages used to generate the navigation.
 export const navigation: NavigationConfig = {
+  help: {
```

Comment:
Also here, I think VSC should be showing an error, and that the `help` section needs to be alphabetical? Let's discuss!